tortoise/fields/data.py

```diff
@@ -713,9 +707,13 @@ def to_db_value(
 IntEnumType = TypeVar("IntEnumType", bound=IntEnum)
 
 
+class IntEnumFieldInstance(IntEnumFieldMixin, SmallIntField):
+    pass
+
+
 def IntEnumField(
```

Comment:
This is really confusing that `IntEnumField` actually returns `SmallIntField` :(

Reply:
Yes, but mostly, IntEnumField used in the following case:
```py
class Gender(IntEnum):
      male = 0
      female = 1
      other = 2

class Users(Model):
    gender = fields.IntEnumField(Gender)
```
And I wish it to be `small int` in db, not int.